subscribie/email.py

```diff
@@ -31,6 +41,13 @@ def send_welcome_email():
     Mail(current_app)
     company = Company.query.first()
     plan = Plan.query.filter_by(uuid=session.get("plan", None)).first()
+    email = session.get("email")
+    ### this part is having problems ### when I put .first() at the end.
```

Comment:
@joeltejeda this may no longer be an error because #557 has been fixed. 
Please re-test this PR , the .first () operation error may not happen now 

I've also forgotten if this PR is still needed , since a different approach it happening on master 

See the conversation tab and related commits

Reply:
@chrisjsimpson  still it's the same error if you change it to  email = session.get("email").first()